### PR TITLE
Ability to exclude client VM

### PR DESCRIPTION
**Pull request by choonchernlim**
The current implementation will always create server VM and client VM.

This PR introduced a new variable `client_enabled` that allows client VM to be excluded.

```
module "forseti" {
  source           = "../"
  domain           = "my-domain"
  project_id       = "project-id"
  org_id           = "1234567890"

  // new variable (type = bool, default = true)
  client_enabled   = false
}
```

Related: #403

### CODE REVIEW

File: test/integration/no_client_vm/controls/shielded_vm.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Copyright 2018 Google LLC
```
Comment (gkowalski-google) [inline test/integration/no_client_vm/controls/shielded_vm.rb]: Nice tests! Fix the file name.

Comment (choonchernlim) [reply test/integration/no_client_vm/controls/shielded_vm.rb]: Fixed

**Comment by choonchernlim** [comment]
@gkowalski-google Can you tell me the CI's detailed errors? I don't have access to the error reports. From the Docker container, it works fine.

```
Profile: no_client_vm
Version: (not specified)
Target:  local://

  ✔  forseti-no-client-vm: Forseti No Client VM
     ✔  Command: `gcloud compute instances list --filter='zone:us-central1-c' --project=ci-forseti-serv-qpyo --format='json(name)'` should have server VM
     ✔  Command: `gcloud compute instances list --filter='zone:us-central1-c' --project=ci-forseti-serv-qpyo --format='json(name)'` should not have client VM
     ✔  Command: `gcloud compute instances list --filter='zone:us-central1-c' --project=ci-forseti-serv-qpyo --format='json(name)'` exit_status should equal 0
     ✔  Command: `gcloud compute instances list --filter='zone:us-central1-c' --project=ci-forseti-serv-qpyo --format='json(name)'` stderr should eq ""
     ✔  Command: `gcloud iam service-accounts list --project=ci-forseti-serv-qpyo --format='json(email)'` should have server service account
     ✔  Command: `gcloud iam service-accounts list --project=ci-forseti-serv-qpyo --format='json(email)'` should not have client service account
     ✔  Command: `gcloud iam service-accounts list --project=ci-forseti-serv-qpyo --format='json(email)'` exit_status should equal 0
     ✔  Command: `gcloud iam service-accounts list --project=ci-forseti-serv-qpyo --format='json(email)'` stderr should eq ""
     ✔  Command: `gsutil ls -p ci-forseti-serv-qpyo` exit_status should eq 0
     ✔  Command: `gsutil ls -p ci-forseti-serv-qpyo` stderr should eq ""
     ✔  Command: `gsutil ls -p ci-forseti-serv-qpyo` stdout should match "gs://forseti-cai-export-3f8b0f4f"
     ✔  Command: `gsutil ls -p ci-forseti-serv-qpyo` stdout should match "gs://forseti-server-3f8b0f4f"
     ✔  Command: `gsutil ls -p ci-forseti-serv-qpyo` stdout should not match "gs://forseti-client-3f8b0f4f"


Profile Summary: 1 successful control, 0 control failures, 0 controls skipped
Test Summary: 13 successful, 0 failures, 0 skipped
```

**Comment by gkowalski-google** [comment]
@choonchernlim Here is the error:

```
module.no-client-vm.module.server.google_compute_firewall.forseti-server-allow-grpc[0]: Creation complete after 15s [id=forseti-server-allow-grpc-c8bd67b0]
       
       Error: Error creating GlobalAddress: googleapi: Error 409: The resource 'projects/ci-forseti-serv-zhwl/global/addresses/private-ip-address' already exists, alreadyExists
       
         on ../../../modules/cloudsql/main.tf line 43, in resource "google_compute_global_address" "private_ip_address":
         43: resource "google_compute_global_address" "private_ip_address" {
```

**Comment by xingao267** [comment]
@choonchernlim Thanks Mike!

**Pull request merged**